index.Rmd

```diff
@@ -65,7 +65,7 @@ band9 <- ("b09.tiff")
 
 
 
-```{r, message=F, warning=F}
+```{r, message=F, warning=F} 
```

Comment:
I got an error on this line that said when I tried to buildthe website:
Error in .rasterObjectFromFile(x, band = band, objecttype = "RasterLayer",  : 
  Cannot create a RasterLayer object from this file. (file does not exist)
Calls: <Anonymous> ... raster -> raster -> .local -> .rasterObjectFromFile
Execution halted

Exited with status 1.

Therefore, your script sadly isn't reproducible yet.